src/components/Highlight/Highlight.tsx

```diff
@@ -1,17 +1,25 @@
 import * as React from 'react';
 import Button from '../../partials/Button';
 
-export interface HighlightProps {}
+export interface HighlightProps {
+  data: {
+    text: string;
+    url: string;
+  };
+}
 
 const Highlight = (props: HighlightProps) => {
+  const {text, url} = props.data;
+  
   return (
     <div className="fullWidthContainer">
       <section className={'highlight'}>
         <div className="container">
           <div className="flexRow flexAlign--center">
             <div className={'highlight__info flexRow'}>
               <img src="/assets/medicon/images/info.png" alt="info" />
-              <p>Poliklinika Zelený pruh je od 16.9. do 19.9. zavřená</p>  
+              {(text && <p>{text}</p>) || 
+                <p>For more info just call us!</p>}  
```

Comment:
why are you using this fallback?, just dont display anything if there is no text, you could add a conddition